tests/gpflow/conditionals/test_multioutput.py

```diff
@@ -23,38 +27,36 @@
 float_type = default_float()
 rng = np.random.RandomState(99201)
 
-
 # ------------------------------------------
 # Helpers
 # ------------------------------------------
 
 
-def predict(model, Xnew, full_cov, full_output_cov):
-    m, v = model.predict_f(Xnew, full_cov=full_cov, full_output_cov=full_output_cov)
-    return [m, v]
-
-
-def predict_all(models, Xnew, full_cov, full_output_cov):
+def predict_all(
+    models: Sequence[SVGP], Xnew: tf.Tensor, full_cov: bool, full_output_cov: bool
+) -> Tuple[List[tf.Tensor], List[tf.Tensor]]:
     """
     Returns the mean and variance of f(Xnew) for each model in `models`.
     """
     ms, vs = [], []
     for model in models:
-        m, v = predict(model, Xnew, full_cov, full_output_cov)
+        m, v = model.predict_f(Xnew, full_cov=full_cov, full_output_cov=full_output_cov)
         ms.append(m)
         vs.append(v)
     return ms, vs
 
 
-def assert_all_array_elements_almost_equal(arr, decimal):
+def assert_all_array_elements_almost_equal(arr: Sequence[tf.Tensor]) -> None:
```

Comment:
Do you think the original intent might have been to set `atol` to `0.1**decimal`?

Reply:
> Do you think the original intent might have been to set `atol` to `0.1**decimal`?

I think the original intent was to do `np.testing.assert_almost_equal(..., decimal=decimal)`.
Apparently that turned into `np.testing.asert_allclose(..., atol=1e-5)`.
:shrug: